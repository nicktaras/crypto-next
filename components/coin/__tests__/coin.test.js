/* eslint-env jest */
import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'
import Coin from '../coin.jsx'

describe('With Snapshot Testing', () => {
  it('should create snapshot of coin', () => {
    const component = renderer.create(<Coin />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
