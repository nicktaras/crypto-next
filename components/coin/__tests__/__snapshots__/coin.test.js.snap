// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing should create snapshot of coin 1`] = `
<div
  className="jsx-2271540330 coin"
>
  <i
    className="jsx-2271540330 cf undefined"
  />
</div>
`;
